targets/web/src/AnimatedStyle.ts

```diff
@@ -164,7 +165,7 @@ export class AnimatedStyle extends AnimatedObject {
     // interpolation of all the props, using the transform functions we defined
     // above
     if (props.length > 0) {
-      style.transform = to(props, (...args) => {
+      style.transform = interpolate(props, (...args) => {
```

Comment:
@aleclarson if I use `to` instead `interpolate` the test fails because `to` is actually `undefined`...? I might need your help on this.

Reply:
It was a circular dependency between `to` and `AnimatedInterpolation` 😄